### PR TITLE
LPS-160818 [Raylife AP] Claim Detail - INSURANCE INFO 

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-160818](https://issues.liferay.com/browse/LPS-160818)

BUG: [LPS-179363](https://issues.liferay.com/browse/LPS-179363)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Review by joanagrego — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/navigation-bar/index.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	align-items: center;
+	background-color: #f3f5f7;
+	border: solid 1px #e7e7ed;
+	border-left-color: transparent;
+	border-right-color: transparent;
+	cursor: pointer;
+	display: flex;
+	height: 49px;
+	margin-top: 10px;
+}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/components/navigation-bar/index.scss:25]: Please change it to the common/styles/components file

**Review by joanagrego — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/Types/index.ts [hunk 2]
```diff
@@ -13,11 +13,12 @@
...
 export type ClaimType = {
-	claimAmount: number;
-	claimCreateDate: string;
-	claimStatus: {key: string; name: string};
-	id: number;
-	r_policyToClaims_c_raylifePolicy: {
+	claimAmount?: number;
+	claimCreateDate?: string;
+	claimStatus?: {key: string; name: string};
+	dataJSON?: any;
```
[the remark this answers is not in the document]
Comment (eduardo-diniz) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/Types/index.ts:19]: I tried, but it's not that type.

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by eduardo-diniz** [comment]
ci:revelant

**Comment by eduardo-diniz** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**